iambic/plugins/v0_1_0/aws/organizations/scp/models.py

```diff
@@ -241,6 +241,9 @@ def sort_tags(cls, v: list[Tag]) -> list[Tag]:
 
 class AwsScpPolicyTemplate(AWSTemplate, AccessModel):
     template_type = AWS_SCP_POLICY_TEMPLATE
+    template_schema_url = (
+        "https://docs.iambic.org/reference/schemas/aws_scp_policy_template"
```

Comment:
url not accessible: maybe that's due to the previous docs didn't include the SCP. 

![no scp yet](https://github.com/noqdev/iambic/assets/1200287/aefd19a6-b0f7-4664-a246-0d04d26a320c)